src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -596,24 +596,50 @@ def computer_test(user, print_traceback, computer):
 
 
 @verdi_computer.command('delete')
+@options.DRY_RUN()
 @arguments.COMPUTER()
 @with_dbenv()
-def computer_delete(computer):
+def computer_delete(computer, dry_run):
     """Delete a computer.
 
-    Note that it is not possible to delete the computer if there are calculations that are using it.
+    Note prompt can delete the computer even if running calculations are using it.
     """
     from aiida import orm
     from aiida.common.exceptions import InvalidOperation
+    from aiida.orm import Computer, Node
+    from aiida.orm.querybuilder import QueryBuilder
+    from aiida.tools import delete_nodes
 
     label = computer.label
 
+    # Sofar, we can only get this info with QueryBuilder
+    builder = QueryBuilder()
+    builder.append(Computer, filters={'label': label}, tag='computer')
+    builder.append(Node, with_computer='computer', project=Node.fields.pk)
+    node_pks = builder.all(flat=True)
+
+    echo.echo_report(f'This computer has {len(node_pks)} associated nodes')
+
+    def _dry_run_callback(pks):
+        if pks:
+            echo.echo_report('The nodes with the following pks would be deleted: ' + ' '.join(map(str, pks)))
+            echo.echo_warning(f'YOU ARE ABOUT TO DELETE {len(pks)} NODES! THIS CANNOT BE UNDONE!')
+        confirm = click.prompt('Shall I continue? [SuRe/N]', type=str) == 'SuRe'
+        if not confirm:
+            raise click.Abort
+        return not confirm
```

Comment:
Why not use `click.confirm` instead? This will automatically abort or return the right value so the code is much simpler. By default it will prompt with `y/N` so if the user just presses enter, the command is aborted. They have to type `y` followed by enter explicitly to continue. I think this is safe enough and is what we use elsewhere across `verdi`.
```suggestion
        return not click.confirm('Are you sure that you want to continue?', abort=True)
```

Reply:
Well, because together with @giovannipizzi and @GeigerJ2  we thought, it might happen that users just confirm everything and pressing `y` without actually reading the message. This way, we wake up users to type something else, instead of `y`, because honestly this special case is really important, it can potentially delete months of calculations, and nothing can get restored if deleted by mistake.

Reply:
Ok, can we at least make it slightly less obnoxious and require `yes`? Forcing the user to type `SuRe` with different capitalization is surely going to annoy users who just want to delete a computer that is of little importance

Reply:
ahahaha, sure :)